content/en/docs/v3.3/installing-on-linux/on-premises/install-kubesphere-on-vmware-vsphere.md

```diff
@@ -422,77 +422,191 @@ spec:
     localVolume:
       storageClassName: local  
 
+---
 ---
 apiVersion: installer.kubesphere.io/v1alpha1
 kind: ClusterConfiguration
 metadata:
   name: ks-installer
   namespace: kubesphere-system
   labels:
-    version: v3.3.0
+    version: master
```

Comment:
Isn't this doc for v3.3.0?

Reply:
> Isn't this doc for v3.3.0?
My mistake. I referenced an incorrect link. 3ks for review. I‘ve updated the doc.